src/__tests__/to-be-disabled.tsx

```diff
@@ -49,6 +49,18 @@ describe('.toBeDisabled', () => {
       expect(() => expect(queryByTestId(name)).not.toBeDisabled()).toThrow();
     });
   });
+
+  test('handle editable prop for TextInput when false', () => {
```

Comment:
```suggestion
  test('handle editable prop for TextInput', () => {
```
you can remove the when true since you handle true and false in this test

Reply:
just like your test for toBeEnabled, you can check the case `<TextInput testID="enabled-by-default" />`